source/algorithms.tex

```diff
@@ -1395,7 +1396,8 @@
 
 \pnum
 The semantics of parallel algorithms invoked with an execution policy object of
-implementation-defined type are implementation-defined.
+\impldef{additional execution policies supported by parallel algorithms} type are
```

Comment:
I assume this form of implementation-defined behavior is introduced elsewhere. Can we check that we use the same index entry for both places?